### ISSUE TITLE
update alt text for images

### ISSUE THREAD

**Issue by malantonio** [LafayetteCollegeLibraries/spot#127]
we should really be applying this to all of the images we have. in a chat with janna, it was recommended that we provide a way for this be an editable field, rather than pulling the title (which may not offer much help when describing some images). 

it might be useful to attach this metadata to the FileSet(s), rather than the work itself (as it describes the File and not necessarily the work), and (somehow) adding a field for editing to the files tab.

presently, hyrax/blacklight uses the image path/query for thumbnails, which seems as unuseful as excluding the field altogether:

```html
<img src="/downloads/2514nk481?file=thumbnail" alt="2514nk481?file=thumbnail" />
```

janna's going to keep her ears open about how other places handle this re: catalog results.

-------

in terms of degrees (for example, none of the to-be-migrated works have alt. text provided):

level               | description
--------------|---------
preferred        | unique alt. text values for each object
acceptable     | using content from a field such as `title`
unacceptable | no content / non-descriptive content (ids)
